src/diffusers/schedulers/scheduling_common_flax.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
Can we maybe add this to `scheduling_utils_flax.py` instead ? :-) We usually don't have `_common_` files in `src/diffusers`

Reply:
Done. I moved it to `scheduling_utils_flax.py`